README.md

```diff
@@ -492,6 +492,120 @@ With the persistence handlers in place, you can now use all persistence methods.
 any or both persistence handlers. If you will only use sync or async persistence, you only need to define the respective
 handler to use these methods.
 
+
+## Create Factory Method
+
+When creating various models you might want to create a Base Factory with Generic Models
```

Comment:
If you prefer to create a factory imperatively, you can do so using the `ModelFactory.create_factory` method. This method receives the following arguments:

- `model`- the model for the factory. 
- `base` - an optional base factory class. Defaults to the factory class on which the method is called. 
- `kwargs` - a dictionary of arguments correlating to the class vars accepted by `ModelFactory`, e.g. `__faker__`.

Reply:
Yeah that is much clearer!